spec/classes/foreman_proxy__plugin__dynflow_spec.rb

```diff
@@ -11,14 +11,14 @@
     end
 
     it { should contain_foreman_proxy__plugin('dynflow') }
-    it 'should configure dynflow.yml' do
-      should contain_file('/etc/foreman-proxy/settings.d/dynflow.yml').
-        with({
-          :ensure  => 'file',
-          :owner   => 'root',
-          :mode    => '0640',
-          :content => /:enabled: https/
-        })
+
+    it 'should generate correct dynflow.yml' do
+      verify_exact_contents(catalogue, "#{etc_dir}/foreman-proxy/settings.d/dynflow.yml", [
```

Comment:
etc_dir is not defined here. I wonder if we can do that in our spec_helper so we don't have to do it in every spec file.